app/app.go

```diff
@@ -246,7 +246,7 @@ type App struct {
 //
 // NOTE: upgradeHeight refers specifically to the height that
 // a node will upgrade from v1 to v2. It will be deprecated in v3
-// in place for a dynamically signalling scheme
+// in place for a dynamically signaling scheme
```

Comment:
also not a typo

```suggestion
// in place for a dynamically signalling scheme
```
<img width="299" alt="Screenshot 2023-12-16 at 2 50 21 PM" src="https://github.com/celestiaorg/celestia-app/assets/3699047/0b6f650b-f823-477b-b142-0d2a0f657f4d">